easybuild/easyconfigs/m/mpi4py/mpi4py-3.1.4-foss-2022b.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
+
+name = 'mpi4py'
+version = '3.1.4'
+
+homepage = 'https://github.com/mpi4py/mpi4py'
+description = """MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for
+ the Python programming language, allowing any Python program to exploit multiple processors."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['17858f2ebc623220d0120d1fa8d428d033dde749c4bc35b33d81a66ad7f93480']
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('SciPy-bundle', '2023.01'),
```

Comment:
As far as I see `mpi4py` does not requires `SciPy-bundle`. `mpi4py` could then go in the `gompi` toolchain.
```suggestion
```